airbyte-integrations/connectors/source-relational-db/src/main/java/io/airbyte/integrations/source/relationaldb/AbstractDbSource.java

```diff
@@ -216,13 +217,16 @@ private void validateCursorFieldForIncrementalTables(
       }
 
       if (!verifyCursorColumnValues(database, stream.getNamespace(), stream.getName(), cursorField.get())) {
-        tablesWithInvalidCursor.add(
+        tablesWithInvalidCursorToWarnAbout.add(
             new InvalidCursorInfo(fullyQualifiedTableName, cursorField.get(),
                 cursorType.toString(), "Cursor column contains NULL value"));
-        continue;
       }
     }
 
+    if (!tablesWithInvalidCursorToWarnAbout.isEmpty()) {
+      LOGGER.warn(InvalidCursorInfoUtil.getInvalidCursorConfigMessage(tablesWithInvalidCursor));
```

Comment:
This is going to break tests

Reply:
ahan! once I will figure out which tests are breaking I will ignore them and add a todo. Does that work?

Reply:
Sure
It should be tableWithNullValueCursorShouldThrowException and createTableWithNullValueCursor in PostgresSourceTest